VideoWeb/VideoWeb/Controllers/VideoEventsController.cs

```diff
@@ -20,7 +20,6 @@ namespace VideoWeb.Controllers
     [ApiController]
     [Route("callback")]
     [Authorize(AuthenticationSchemes = "Callback")]
-    [AllowAnonymous]
```

Comment:
this was AllowAnonymous in master?

Reply:
recently added, correct to remove
https://github.com/hmcts/vh-video-web/pull/1191/files#r578500443